data.md

```diff
@@ -670,8 +670,8 @@ We are using the polymorphic `Map` sort for these word maps.
 ```k
     syntax Int ::= #lookup ( Map , Int ) [function]
  // -----------------------------------------------
-    rule #lookup( (KEY |-> VAL) M, KEY ) => VAL
-    rule #lookup(               M, KEY ) => 0 requires notBool KEY in_keys(M)
+    rule #lookup( (KEY |-> VAL) M, KEY ) => VAL                               [concrete]
```

Comment:
These `[concrete]` attributes are breaking all of the proofs we are working with. It seems we just have to copy over all of these rules into our lemmas with the `[concrete]` tag removed. I'm kinda confused why you are doing this. Do we need to split up evm-semantics into a symbolic and concrete version?

Reply:
We had to do the same in verified-smart-contracts. Look at our old projects like erc20/hkg for model. If you use precise gas computation for loops, then take bihu as model.

Reply:
@MrChico Let us fix this problem.  I think we can find a way without this change.

Reply:
We will have a follow up PR into K which adds some flags which allow us to specify that we want certain rules to be concrete at `kprove` time.

Reply:
So with that PR we would remove these `[concrete]` flags again from this repo? I guess we'll just wait for that to get merged then

Reply:
Yep.